src/test/java/org/broadinstitute/hellbender/tools/dataflow/pipelines/BaseRecalibratorDataflowIntegrationTest.java

```diff
@@ -111,11 +111,13 @@ public void runAllTests() throws Exception {
         final String hg19Ref = "EMWV_ZfLxrDY-wE";
         final String GRCh37Ref = RefAPISource.GRCH37_REF_ID;
         final String HiSeqBam = localResources + "CEUTrio.HiSeq.WGS.b37.ch20.1m-1m1k.NA12878.bam";
+        final String trickyBam = localResources + "CEUTrio.HiSeq.WGS.b37.ch20.4379150-4379157.bam";
```

Comment:
It's called tricky because that's the one that made even BaseRecalibrator give wrong answers.
See https://github.com/broadinstitute/hellbender/pull/878 for the full background.